package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "paella-engage-ui",
-  "version": "4.0.10",
+  "version": "4.0.11",
```

Comment:
You can, but it doesn't matter – shrinkwrap is concerned the dependencies of paella-matterhorn rather than paella-matterhorn itself.